### PR TITLE
Split oidc_provider

### PR DESCRIPTION
**Pull request by olemarkus**
* one builder concerned with publishing issuer discovery metadata
* one builder concerned with creating aws oidc provider

### CODE REVIEW

**Review by hakman — APPROVED** [review]

**Review by hakman — APPROVED** [review]

File: pkg/model/BUILD.bazel [hunk 1]
```diff
@@ -59,7 +61,6 @@ go_library(
 go_test(
     name = "go_default_test",
     srcs = ["bootstrapscript_test.go"],
-    data = glob(["tests/**"]),  #keep
```
Comment (hakman) [inline pkg/model/BUILD.bazel:62]: This has to be added back :)

Comment (johngmyers) [reply pkg/model/BUILD.bazel:62]: But verify-bazel passed?

Comment (hakman) [reply pkg/model/BUILD.bazel:62]: This is manually added and cannot be verified by that, if I recall correctly.

**Pull request merged**